src/review.py

```diff
@@ -1,25 +1,53 @@
 import automatic_code_review_commons as commons
-
+import os
+import re
 def review(config):
-    path_target = config['path_target']
-    path_source = config['path_source']
-
     merge = config['merge']
-    project_id = merge['project_id']
-    merge_request_id = merge['merge_request_id']
-    
+    wrong_pattern = config['wrongPattern']
+    rigth_pattern = config['rigthPattern']
+    regexList = config['regexFile']
+    changes = merge['changes']
+
     comments = []
-    
-    # TODO IMPLEMENTAR EXTENSION
-    #  O OBJETO DE COMENTARIO DEVE POSSUIR O SEGUINTE FORMATO
-    #  commons.comment_create(
-    #     comment_id=commons.comment_generate_id( "" ),
-    #     comment_path="",
-    #     comment_description="",
-    #     comment_snipset=True,
-    #     comment_end_line=1,
-    #     comment_start_line=1,
-    #     comment_language="",
-    # )
+
+    filesForRegex = []
+    for change in changes:
+        if re.match(regexList[0], getFileName(change.get('new_path'))):
+            #TODO verificar se o arquivo vai vir no path_source ou new_path
+            filesForRegex.append(change.get('new_path'))
+
+    lines_to_comment = findWrongPattern(filesForRegex, wrong_pattern)
+
+    for key, value in lines_to_comment.items():
+        descr_comment = config['message']
+        descr_comment = descr_comment.replace("${LINE_NUMBER}", str(key))
+        descr_comment = descr_comment.replace("${WRONG_PATTERN}", value.replace("\n", ""))
+        descr_comment = descr_comment.replace("${RIGHT_PATTERN}", rigth_pattern)
+
+        comments.append(commons.comment_create(
+            comment_id=commons.comment_generate_id(descr_comment),
```

Comment:
pra gerar o id daria para colocar path+line<br>isso pq se a mensagem customizada for generica, pode duplicar o id, e como a mensagem vem de uma config melhor usarmos o path e line

Reply:
aqui ficou faltando o path... a ideia era ser path+line, e ta comentario+line, ainda pode duplicar